Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -727,7 +727,7 @@ static bool isRenderableTextureView(const TextureView& texture)
     auto mtlRenderCommandEncoder = [m_commandBuffer renderCommandEncoderWithDescriptor:mtlDescriptor];
     ASSERT(!m_existingCommandEncoder);
     setExistingEncoder(mtlRenderCommandEncoder);
-    return RenderPassEncoder::create(mtlRenderCommandEncoder, descriptor, visibilityResultBufferSize, depthReadOnly, stencilReadOnly, *this, visibilityResultBuffer, maxDrawCount, m_device);
+    return RenderPassEncoder::create(mtlRenderCommandEncoder, descriptor, visibilityResultBufferSize, depthReadOnly, stencilReadOnly, *this, visibilityResultBuffer, maxDrawCount, mtlDescriptor.stencilAttachment.texture.pixelFormat == MTLPixelFormatX32_Stencil8, m_device);
```

Comment:
I'm just curious, but would it make more sense to pass the pixelFormat instead of a boolean?

Reply:
oh that's a good idea, but actually I realized we already have this information, I don't need to pass another parameter.

On RenderPassEncoder.cpp:135 we have:
```
id<MTLTexture> depthTexture = textureView.texture();
```
I can get the pixel format from there

Reply:
PR is updated and much simpler now :)